libraries/apollo-annotations/src/commonMain/kotlin/com/apollographql/apollo3/annotations/ApolloEnumConstructor.kt

```diff
@@ -0,0 +1,9 @@
+package com.apollographql.apollo3.annotations
+
+@RequiresOptIn(
+    level = RequiresOptIn.Level.ERROR,
+    message = "Applied to the constructor of an enum's UNKNOWN__ value, as instantiating it is usually a mistake."
```

Comment:
```suggestion
    message = "The `UNKNOWN__` class represents GraphQL enums that are not present in the schema and whose `rawValue` cannot be checked at build time. You may want to update your schema instead of calling this constructor directly."
```

Reply:
A lot nicer! 🙏